shor/transpilers/qiskit.py

```diff
@@ -6,9 +6,11 @@
 
 
 def to_qiskit_circuit(shor_circuit: QC) -> QuantumCircuit:
+    qbit_registers, cbit_registers = shor_circuit.to_registers()
+
     qiskit_circuit = QuantumCircuit(
-        len(shor_circuit.initial_state()),
-        len(shor_circuit.measure_bits()),
+        len(qbit_registers._qbits),
```

Comment:
What this is doing is concatenating all of the qbit registers, and cbit registers respectively.
And this is allocating a qiskit register of a matching length for each.

We are still relying on relative indexing of circuit in our unit tests, so this should work for basic use cases

Reply:
prefer not accessing the _qbits directly.

Reply:
Can just override the length built in function

Reply:
its not exactly clear here what your changing/ want to be there. 
The first two calls make sense as len(shor_circuit.initial_state()) gives number of qubits and len(shor_circuit.measure_bits()) gives number of bits the you intend to measure--though I think it makes more sense to initialize the circuit with the number of cbits not the number you intend to measure
so sorry what are you looking for here then?

Reply:
Im also a little confused on what your changing here and what was already there. The syntax Ive been using its 
    qiskit_circuit = QuantumCircuit(
        len(qbit_registers._qbits),
        len(cbit_registers._cbits),
    )
In a previous review message I thought you said that i should make a new qubit/cbit class to keep track of the number

Reply:
Unless your changing my code to 
qiskit_circuit = QuantumCircuit(
        len(shor_circuit.initial_state()),
        len(shor_circuit.measure_bits()),
        len(qbit_registers._qbits),

For some reason there are issues with git not accessing my up to date version